certbot-apache/certbot_apache/_internal/parser.py

```diff
@@ -47,6 +47,7 @@ class ApacheParser:
     arg_var_interpreter: Pattern = re.compile(r"\$\{[^ \}]*}")
     fnmatch_chars: Set[str] = {"*", "?", "\\", "[", "]"}
 
+    # pylint: disable=unused-argument
```

Comment:
I mean... if we're doing this for 2.0 anyway, we can just remove `version`.

Reply:
I did try removing `version` entirely and things got a little bit curly due to how extensively it is used throughout `certbot-apache`. I can give it another go, but we can also do it another time.

Reply:
Oh for reason when looking at this I convinced myself that this was a public signature but we're in `_internal` here so yeah definitely good with leaving this for now.